source/reviews/standard.txt

```diff
@@ -0,0 +1,90 @@
+======================
+The Standard of Review
+======================
+
+.. _pr-standard:
+
+Purpose
+-------
+
+The primary purpose of a review is to make sure that the overall health of MongoDB's
+corpus is improving over time. All of the tools and processes of review are
+designed to this end.
+
+In order to accomplish this, a series of trade-offs have to be balanced.
+
+First, authors must be able to **make progress** on their tasks. If you never
+submit an improvement to the corpus, then the corpus never improves. Also,
```

Comment:
Question: Could "make progress" be defined? I'm also not sure what "if you never submit an improvement..." means in this context. Does it relate to "making progress"? Is this saying, "approved PRs are the way the MongoDB collection of writings improves, and thus you should favor approving PRs"?